network-area-diagram/src/main/java/com/powsybl/nad/layout/BasicFixedLayoutFactory.java

```diff
@@ -29,4 +29,15 @@ public Layout create() {
         layout.setFixedNodePositions(fixedPositions);
         return layout;
     }
+
+    public Layout create(LayoutFactory layoutFactory) {
```

Comment:
We'd like to have this part of code inside `public Layout create()` instead so that it gets called in `NetworkAreaDiagram::draw`.